test/pachyderm_helm_test.go

```diff
@@ -75,10 +75,14 @@ func TestPachdImageTag(t *testing.T) {
 func TestPachdImageTagDeploymentEnv(t *testing.T) {
 	helmChartPath := "../pachyderm"
 
-	expectedTag := "1234"
-	expectedContainerImage := "pachyderm/pachd:" + expectedTag
+	expectedTag := "blah1234"
```

Comment:
Any reason to change it from `"1234"`? An all-digit tag is allowed.

No real reason _not_ to change it, either. Just curious!

Reply:
forthcomming json validation doesn't accept string (I guess it could though)